### DISCUSSION TITLE
Are the in-app purchases documented somewhere?

### DISCUSSION THREAD

**Discussion by snoblenet** [rxhanson/Charmstone-Community#38] [General]
On The App Store, it says Charmstone has in-app purchases.

If the app is good, I'm willing to pay real money to buy it outright, a smaller amount of real money for functionality upgrades, and a smaller amount of real money for compatibility upgrades with future Mac OS X releases. (Security upgrades should be free.)

But I'm not willing to subscribe to something that is not a service and I'm worried that the required purchases could change in the future.

Can you clarify? Thanks.

**Comment by rxhanson** [comment]
Thanks for checking out Charmstone. The "full version" in-app purchase is to enable the four corner positions and the "far" positions (for when you move the cursor farther away). If you purchase the "full version", then it's yours to keep. I don't have any plans to create any other in-app purchases, so there's nothing hidden here. If for some reason I were to add new features at some point, I'd likely just add them into the full version, but I at the moment view Charmstone as "feature complete". If for some reason I did make another version and treated it as an upgrade, then I'd make sure that an upgrade path was clear and fair. Does that answer your question?
